### ISSUE TITLE
[Help Wanted] [PLT-7693] 508 Compliance: After saving an account setting, set focus to the next setting on the list

### ISSUE THREAD

**Issue by esethna** [mattermost/mattermost-server#8032]
If you're interested please comment here and come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://pre-release.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://docs.mattermost.com/guides/developer.html), specifically for [machine setup](https://docs.mattermost.com/developer/developer-setup.html) and for [developer workflow](https://docs.mattermost.com/developer/developer-flow.html).

----

**Notes**: [Jira ticket](https://mattermost.atlassian.net/browse/PLT-7693)

**Comment by santos22** [comment]
Are **[refs](https://reactjs.org/docs/refs-and-the-dom.html)** the way to go regarding this issue? 🤔

**Comment by saturninoabril** [comment]
@santos22 Yes, it could be. Another way is passing additional props to the SettingItemMin component where it will set the focus to the Edit button (similar to above PR).
Looks like @liusy182 worked on this issue already :)
